subjects/union.en.md

```diff
@@ -12,15 +12,15 @@ If the number of arguments is different from 2, then the program displays newlin
 
 ```console
 student@ubuntu:~/[[ROOT]]/union$ go build
-student@ubuntu:~/[[ROOT]]/union$ ./union "zpadinton" "paqefwtdjetyiytjneytjoeyjnejeyj" | cat -e
+student@ubuntu:~/[[ROOT]]/union$ ./union zpadinton paqefwtdjetyiytjneytjoeyjnejeyj | cat -e
 zpadintoqefwjy$
-student@ubuntu:~/[[ROOT]]/union$ ./union "ddf6vewg64f" "gtwthgdwthdwfteewhrtag6h4ffdhsd" | cat -e
+student@ubuntu:~/[[ROOT]]/union$ ./union ddf6vewg64f gtwthgdwthdwfteewhrtag6h4ffdhsd | cat -e
 df6vewg4thras$
-student@ubuntu:~/[[ROOT]]/union$ ./union "rien" "cette phrase ne cache rien" | cat -e
-rienct phas$
+student@ubuntu:~/[[ROOT]]/union$ ./union rien cettephrasenecacherien | cat -e
```

Comment:
here you need the quotes (you can write it this way but it is less readable : `cette\ phrase\ ne\ cache\ rien`)